ckanext/stcndm/bin/massage_imdb.py

```diff
@@ -0,0 +1,124 @@
+__author__ = 'marc'
+
+import sys
+import json
+import yaml
+# import commands
+import subprocess
+
+
+def run_command(command):
+    p = subprocess.Popen(command,
+                         stdout=subprocess.PIPE)
+    return iter(p.stdout.readline, b'')
+
+
+def code_lookup(old_field_name, data_set, choice_list):
+    temp = data_set[old_field_name]
+    if isinstance(temp, unicode):
+        field_values = map(unicode.strip, temp.split(';'))
+    else:
+        field_values = map(unicode.strip, temp[0].split(';'))
+    codes = []
+    for field_value in field_values:
+        code = None
+        for choice in choice_list:
+            if choice['label']['en'] == field_value:
+                if choice['value']:
+                    code = choice['value']
+        if not code:
+            print 'weird '+old_field_name+' .'+data_set[old_field_name]+'.'+field_value+'.'
+        else:
+            codes.append(code)
+    return codes
+
+content_type_list = []
+for line in run_command('curl http://127.0.0.1:5000/api/3/action/package_search?q=type:codeset&rows=100'.split()):
+    raw_codesets = json.loads(line)['result']['results']
+    content_type_list = []
+    for codeset in raw_codesets:
+        if codeset['codeset_type'] == 'content_type':
+            content_type_list.append({
+                'label': codeset['title'],
+                'value': codeset['codeset_value']
+            })
```

Comment:
You could avoid shelling out to curl by using [ckanapi.RemoteCKAN](https://github.com/ckan/ckanapi#remoteckan) just like: https://github.com/open-data/ckanext-stcndm/blob/master/ckanext/stcndm/helpers.py#L248-L253